### PR TITLE
feat: #1014 Inline help rebuild with popover component

### PR DESCRIPTION
**Pull request by athulvraj**
## Related Issue
Closes SAP/fundamental-styles#

## Description
Rebuilding the InlineHelp component with Popover component 

#### Please check whether the PR fulfills the following requirements

- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-styles/wiki/PR-Review-Checklist

### CODE REVIEW

File: docs/pages/components/inline-help.md [hunk 1]
```diff
@@ -17,25 +17,33 @@ Inline help is used to display help text in a popover, often inline with headers
 ## Default Position
 The default positioning of inline help component is bottom right.
 {% capture inline-help %}
-
-<span class="fd-inline-help">
-    <span class="fd-inline-help__content fd-inline-help__content--bottom-right">
-        Lorem ipsum dolor sit amet, consectetur adipiscing.
-    </span>
-</span>
+<div class="fd-popover">
```
Comment (droshev) [inline docs/pages/components/inline-help.md:20]: I think this PR should be `feat:` as it brings changes in the markup

File: src/inline-help.scss [hunk 2]
```diff
@@ -69,161 +68,7 @@ $block: #{$fd-namespace}-inline-help;
...
-      &::before {
-        top: -($fd-tooltip-arrow-offset);
-        right: $fd-tooltip-arrow-offset * 20.5;
-      }
-
-      &::after {
-        top: -($fd-tooltip-arrow-offset - 1);
-        right: $fd-tooltip-arrow-offset * 20.5;
-      }
-    }
```
Comment (droshev) [inline src/inline-help.scss:226]: If you are migrating the component then you should get rid of the old variables `fd-` and use the theming parameters

Comment (jbadan) [reply src/inline-help.scss:226]: @athulvraj ping me if you need help with the theming parameters

Comment (athulvraj) [reply src/inline-help.scss:226]: Sure, I will do

File: docs/pages/components/inline-help.md (hunk 1 above)
Comment (athulvraj) [reply docs/pages/components/inline-help.md:20]: Done

**Comment by JKMarkowski** [comment]
Arrow on position examples is not centered 
image
Also let's get rid of old variables and put theming parameters.

File: test/resources/layout.css [hunk 3]
```diff
@@ -62,6 +62,7 @@ body {
 
 .btn-playground,
 .btn-playground--compact,
+.btn-playground--transparent,
 .btn-playground--emphasized {
   border: 1px solid black;
   border-radius: 4px;
```
Comment (rengare) [inline test/resources/layout.css:68]: `rem` value should be used instead of `px`. There are other places with the same issue

**Comment by stefanoScalzo** [comment]
There are other files, like tile that should not be in this PR, please remove them

**Comment by athulvraj** [comment]
> There are other files, like tile that should not be in this PR, please remove them

Yes, looks like the PR is corrupted.  Will close this and create a new one

**Pull request closed without merging**